### ISSUE TITLE
Grant Super Admin Access to All Spaces

### ISSUE THREAD

**Issue by veronika-teknika** [docmost/docmost#532]
Currently, Super Admins do not have automatic access to all Spaces, which can limit their ability to manage and oversee them effectively. We need to ensure that Super Admins have default access to every Space across the platform, regardless of individual Space settings.

Acceptance Criteria:

-Super Admins are granted automatic access to all existing and newly created Spaces.
-This access cannot be revoked by individual Space settings.
-The UI reflects that Super Admins have full access rights.

Thank you!!!

**Comment by enchove** [comment]
That's a good idea.
But it may violate someone's privacy.

I think if a Super User has audit permissions, it should be clear to the user that they can audit, and a log should be left so that the user knows when and how the audit was performed.

Perhaps this could be replaced with a Group (or Team) leader feature.
Only members of that group can be audited by the Group Leader.

You could also implement a way to categorize team spaces and personal spaces, like Notion does, to preserve individual privacy.

It would be nice to be able to clearly categorize personal and shared pages on the main page.

Thanks

**Comment by prasanna-balaji-sbna** [comment]
Superadmin is the person who might want all the access in the organization. In a company using docmost the superadmin should see all the spaces. so he might change or manage the employees have access to the documents as they are company assets. Private spaces can have their usernames as a single private space for the user. Team space should not be under any specific user.

**Comment by samuelthng** [comment]
I agree with @enchove - we don't need (and do not want) the super admin to see any of the contents, but would be great to still have management permissions over the spaces.

Eg: For IT dept - they don't need to know what Marketing has in their space, nor should they be able to delete or edit anything.
